src/SumoCoders/FrameworkCoreBundle/Service/KeyValueStore.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Service;
+
+/**
+ * Class KeyValueStore
+ *
+ * @package SumoCoders\FrameworkCoreBundle\Service
+ */
+class KeyValueStore
```

Comment:
Can't you use Symfony's ParameterBag class instead of a custom KeyValueStore implementation?